apps/core/components/rangeDatePicker/RangeDatePickerTypes.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import * as React from 'react';
+
+export type Props = {|
+  +isVisible: boolean,
+  +mode?: 'date' | 'time' | 'datetime',
+  +datePickerMode?: 'calendar' | 'spinner' | 'default', // this prop is supported only on android
```

Comment:
This one is not and probably will not be used.

Reply:
Removed